.github/scripts/build

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+usage() {
+    echo``
+    echo "build 'neuralmagic-vllm' in specified python and venv"
+    echo
+    echo "usage: ${0} <options>"
+    echo
+    echo "  -a    - pypi server address (optional)"
+    echo "  -p    - python version"
+    echo "  -v    - name for virtualenv"
+    echo "  -h    - this list of options"
+    echo
+}
+
+PYPI_IP=192.168.201.226
```

Comment:
is this script going to end up in the public repo?  not a big deal for now but we might want to remove this ip addr.

Reply:
maybe we can add it as a secret to GHA?

Reply:
yeah, agree ... i'll update and remove it ...